### ISSUE TITLE
"Started Application" appears twice when Spring Cloud bound

### ISSUE THREAD

**Issue by rnavarropiris** [spring-cloud/spring-cloud-config#403]
## Problem description

If the `org.springframework.cloud:spring-cloud-starter-config` artifact is bound, the log message "Started Application in X seconds" appears twice, the first one being too early (the context was not fully initialized yet).

## Problem reproduction

Using the Spring Boot [Getting started application](https://spring.io/guides/gs/spring-boot):
* execute the application (`complete`) as provided, "Started Application" logged once
* add this dependency (`compile('org.springframework.cloud:spring-cloud-starter-config:1.0.6.RELEASE')`) to the `./gs-spring-boot/complete/build.gradle` dependencies
* execute the application again, "Started Application" appears too early, further initialisations occur, "Started Application" is logged again (this time correctly)

It seems as if the Cloud layer initializes as a separate application. However, the main class is still logged as `hello.Application`, which doesn't make any sense

## Log output

```
2016-05-18 14:30:15.915  INFO 38193 --- [           main] hello.Application                        : Starting Application on RubensMac with PID 38193 (/Users/ruben/tmp/gs-spring-boot/complete/build/libs/gs-spring-boot-0.1.0.jar started by ruben in /Users/ruben/tmp/gs-spring-boot/complete)
2016-05-18 14:30:15.921  INFO 38193 --- [           main] hello.Application                        : No active profile set, falling back to default profiles: default
2016-05-18 14:30:15.966  INFO 38193 --- [           main] s.c.a.AnnotationConfigApplicationContext : Refreshing org.springframework.context.annotation.AnnotationConfigApplicationContext@290e7ede: startup date [Wed May 18 14:30:15 CEST 2016]; root of context hierarchy
2016-05-18 14:30:16.379  INFO 38193 --- [           main] trationDelegate$BeanPostProcessorChecker : Bean 'encrypt.CONFIGURATION_PROPERTIES' of type [class org.springframework.cloud.bootstrap.encrypt.KeyProperties] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
2016-05-18 14:30:16.380  INFO 38193 --- [           main] trationDelegate$BeanPostProcessorChecker : Bean 'encryptionBootstrapConfiguration' of type [class org.springframework.cloud.bootstrap.encrypt.EncryptionBootstrapConfiguration$$EnhancerBySpringCGLIB$$bc8c321a] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
2016-05-18 14:30:16.475  INFO 38193 --- [           main] hello.Application                        : Started Application in 0.766 seconds (JVM running for 1.152)

  .   ____          _            __ _ _
 /\\ / ___'_ __ _ _(_)_ __  __ _ \ \ \ \
( ( )\___ | '_ | '_| | '_ \/ _` | \ \ \ \
 \\/  ___)| |_)| | | | | || (_| |  ) ) ) )
  '  |____| .__|_| |_|_| |_\__, | / / / /
 =========|_|==============|___/=/_/_/_/
 :: Spring Boot ::        (v1.3.5.RELEASE)

2016-05-18 14:30:16.699  WARN 38193 --- [           main] c.c.c.ConfigServicePropertySourceLocator : Could not locate PropertySource: I/O error on GET request for "http://localhost:8888/bootstrap/default": Connection refused; nested exception is java.net.ConnectException: Connection refused
2016-05-18 14:30:16.699  INFO 38193 --- [           main] hello.Application                        : No active profile set, falling back to default profiles: default
2016-05-18 14:30:16.710  INFO 38193 --- [           main] ationConfigEmbeddedWebApplicationContext : Refreshing org.springframework.boot.context.embedded.AnnotationConfigEmbeddedWebApplicationContext@794fe0c2: startup date [Wed May 18 14:30:16 CEST 2016]; parent: org.springframework.context.annotation.AnnotationConfigApplicationContext@290e7ede
...
2016-05-18 14:30:18.780  INFO 38193 --- [           main] s.b.c.e.t.TomcatEmbeddedServletContainer : Tomcat started on port(s): 8080 (http)
2016-05-18 14:30:18.783  INFO 38193 --- [           main] hello.Application                        : Started Application in 3.135 seconds (JVM running for 3.459)
```

**Comment by rnavarropiris** [comment]
I just tried again with the latest version (`compile('org.springframework.cloud:spring-cloud-starter-config:1.1.0.RELEASE')`) and the problem is gone

**Comment by shakuzen** [comment]
I haven't looked at the reproduction repository, but from the description and the logs, I am guessing this was fixed by spring-cloud/spring-cloud-commons#100 in Brixton.RC2

**Comment by dsyer** [comment]
@shakuzen probably. It's normal anyway (switching off the logging didn't change the fact that there are 2 application contexts).
